src/cz/crcs/ectester/standalone/ECTesterStandalone.java

```diff
@@ -736,8 +737,6 @@ private void test() throws TestException, ParserConfigurationException {
                 break;
         }
 
-        String suiteName = cli.getArg(0);
```

Comment:
Should this be removed?

Reply:
It's just something I stumbled upon when debugging. The variable is not used anywhere.